easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.2-GCC-10.2.0.eb

```diff
@@ -0,0 +1,37 @@
+name = 'OpenMPI'
+version = '4.1.2'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['9b78c7cf7fc32131c5cf43dd2ab9740149d9d87cadb2e2189f02685749a6b527']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('hwloc', '2.2.0'),
+    ('libevent', '2.1.12'),
+    ('UCX', '1.9.0'),
+    ('libfabric', '1.11.0'),
+    ('PMIx', '3.1.5'),
+]
+
+# replace -march=skylake-avx512 option with equivalent -m* flags,
+# to avoid problems when building on systems which don't support AVX-512;
+# cfr. https://github.com/open-mpi/ompi/issues/8306
+preconfigopts = "sed -i 's/-march=skylake-avx512/-mavx512f -mavx512bw -mavx512vl -mavx512dq/g' configure && "
```

Comment:
@kosl We don't have this in the OpenMPI 4.1.1 easyconfig (which works fine on Intel Haswell and AMD Rome, where AVX-512 is not supported), so it's not needed at all anymore for OpenMPI 4.1.2 either (only for OpenMPI 4.1.0), so please remove this block.